NGitLab.Mock/Repository.cs

```diff
@@ -323,7 +323,7 @@ public IEnumerable<Commit> GetBranchCommits(string branchName)
             }
         }
 
-        public IEnumerable<Commit> GetCommits(string @ref)
+        public IEnumerable<Commit> GetCommits(string @ref = "")
```

Comment:
I think this is a breaking change (the CI will confirm :)). You need to create a new overload without the parameter. Also, prefer `null` to `""`

Reply:
Got it, I was not sure if it was better to add a new overload, i'll add one!